api/proto/teleport/okta/v1/okta_service.proto

```diff
@@ -150,14 +150,10 @@ message UpdateIntegrationRequest {
   bool enable_app_group_sync = 6;
   // Settings related to access list synchronization.
   AccessListSettings access_list_settings = 7;
-  // Option to reuse an existing connector.
-  string reuse_connector = 8;
-  // SSO metadata URL for integration.
-  string sso_metadata_url = 9;
   // Enable sync from Teleport to Okta. If disabled it will prevent creating Okta assignments which
   // in turn will disable JIT Access Requests. Makes sense only when apps and groups sync is
   // enabled.
-  bool enable_bidirectional_sync = 10;
+  bool enable_bidirectional_sync = 8;
```

Comment:
This is a breaking change isn't it?

Reply:
As I wrote in the description, compatibility is broken, but this request is not yet used so it isn't a problem.

Reply:
As I wrote in the description, compatibility is broken, but this request is not yet used so it isn't a problem.